dataland-frontend/tests/e2e/specs/quality-assurance/CheckQaProcedure.ts

```diff
@@ -35,41 +29,14 @@ describeIf(
         DataTypeEnum.EutaxonomyFinancials,
         testCompany,
         testData.t,
-        fillEuTaxonomyForm,
+        (data) => fillAndValidateEuTaxonomyCreditInstitutionForm(data),
         (req) => (req.headers["REQUIRE-QA"] = "true"),
-        () => testSubmittedDatasetIsInReviewList(companyName)
+        () => testSubmittedDatasetIsInReviewList(testCompany.companyName)
       );
     });
   }
 );
 
-/**
- * Fills the eutaxonomy-financials upload form with the given dataset
- * @param data the data to fill the form with
- */
-function fillEuTaxonomyForm(data: EuTaxonomyDataForFinancials): void {
-  fillEuTaxonomyForFinancialsRequiredFields(data);
-
-  cy.get('[data-test="MultiSelectfinancialServicesTypes"]')
-    .click()
-    .get("div.p-multiselect-panel")
-    .find("li.p-multiselect-item")
-    .first()
-    .click({ force: true });
-
-  cy.get('[data-test="addKpisButton"]').click({ force: true });
-
-  fillEligibilityKpis("creditInstitutionKpis", data.eligibilityKpis?.CreditInstitution);
-  fillField(
-    "creditInstitutionKpis",
-    "tradingPortfolioAndInterbankLoans",
-    data.creditInstitutionKpis?.tradingPortfolioAndInterbankLoans
-  );
-  fillField("creditInstitutionKpis", "tradingPortfolio", data.creditInstitutionKpis?.tradingPortfolio);
-  fillField("creditInstitutionKpis", "interbankLoans", data.creditInstitutionKpis?.interbankLoans);
-  fillField("creditInstitutionKpis", "greenAssetRatio", data.creditInstitutionKpis?.greenAssetRatio);
-}
-
```

Comment:
You have moved this function somewhere else right?
Why?

Reply:
Code duplication. There was another function that did the same or similar.